index.html

```diff
@@ -0,0 +1,449 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <div class="menu-bars">
+        <i class="fas fa-bars open-btn"></i>
+        <i class="fas fa-times close-btn"></i>
+      </div>
```

Comment:
![comparison2](https://user-images.githubusercontent.com/49184579/175209542-134ccb37-e5ff-4521-bf8b-ea6296def404.png)
1. Please follow the Figma Design. The menu options' font, margin, and padding are not looking similar.
2. Your hamburger button is visible and working, but the close (X) button is also visible alongside the hamburger button. Please remove the close (X) button when the hamburger button is visible.

Reply:
Changes are done:+1: